docs/core/deploying/native-aot.md

```diff
@@ -18,41 +18,41 @@ There are some limitations in the .NET native AOT deployment model, with the mai
 > [!WARNING]
 > Native AOT is supported in .NET 7 but only a limited number of libraries are fully compatible with native AOT in .NET 7.
 
-## Publish Native AOT - CLI
-
-Publish a native AOT application using the [dotnet publish](../tools/dotnet-publish.md) command.
-
-01. Add `<PublishAot>true</PublishAot>` to your project file.
-
-    This will produce a native AOT app and shows PublishAot compatibility warnings during build.
-
-    ```xml
-    <PropertyGroup>
-        <PublishAot>true</PublishAot>
-    </PropertyGroup>
-    ```
+## Prerequisites
 
-02. Ensure pre-requisites that are needed for publishing a native AOT application are in the machine.
+The following prerequisites need to be installed before publishing .NET projects with native AOT.
 
-    On Windows, install [Visual Studio 2022](https://visualstudio.microsoft.com/vs/), including Desktop development with C++ workload.
+On Windows, install [Visual Studio 2022](https://visualstudio.microsoft.com/vs/), including Desktop development with C++ workload.
 
-    On Linux, install clang and developer packages for libraries that .NET runtime depends on.
+On Linux, install clang and developer packages for libraries that .NET runtime depends on.
 
-    *Ubuntu (18.04+)*
+- Ubuntu (18.04+)
 
     ```sh
     sudo apt-get install clang zlib1g-dev
     ```
 
-    *Alpine (3.15+)*
+- Alpine (3.15+)
 
     ```sh
     sudo apk add clang gcc lld musl-dev build-base zlib-dev
     ```
 
-03. Publish the app for a specific runtime identifier using `dotnet publish -r <RID>`.
+## Publish Native AOT - CLI
 
-    The following example publishes the app for Windows as a native AOT application on a machine with the required pre-requisites installed.
+01. Add `<PublishAot>true</PublishAot>` to your project file.
```

Comment:
Should we add this property to the MSBuild reference at https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#compilation-related-properties?

Reply:
I do not think it makes sense to document all properties one can possible use in .csproj in one giant file.

https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#compilation-related-properties is missing number of properties. For example, it is missing `PublishReadyToRun` and number of other properties documented in https://docs.microsoft.com/en-us/dotnet/core/deploying/ready-to-run; or it is missing `PublishTrimmed` and number of other properties documented in https://docs.microsoft.com/en-us/dotnet/core/deploying/trimming/trimming-options .